pallets/usage/src/tests.rs

```diff
@@ -630,3 +661,77 @@ fn dust_discarded_with_noop() {
         assert_eq!(Balances::free_balance(&11), 0);
     });
 }
+
+#[test]
+fn gas_properly_handled_for_trap_replies() {
+    init_logger();
+    new_test_ext().execute_with(|| {
+        // 1st user has just above `T::TrapReplyExistentialGasLimit` reserved
+        assert_ok!(<Balances as ReservableCurrency<_>>::reserve(&1, 1_100));
+        // 2nd user already has less than `T::TrapReplyExistentialGasLimit` reserved
+        assert_ok!(<Balances as ReservableCurrency<_>>::reserve(&2, 500));
+
+        run_to_block(10);
+
+        // Populate wait list with 2 messages
+        populate_wait_list_with_split(2, 10, 2, 1_100);
+
+        let wl = wait_list_contents()
+            .into_iter()
+            .map(|(d, n)| (d.message().clone(), n))
+            .collect::<Vec<_>>();
+        assert_eq!(wl.len(), 2);
+
+        // Insert respective programs to the program storage
+        let program_1 = gear_core::program::Program::new(
+            1.into(),
+            CheckedCode::try_new(
+                hex!("0061736d01000000020f0103656e76066d656d6f7279020001").to_vec(),
+            )
+            .unwrap(),
+        );
+        crate::mock::set_program(program_1);
+
+        let program_2 = gear_core::program::Program::new(
+            2.into(),
+            CheckedCode::try_new(
+                hex!["0061736d01000000020f0103656e76066d656d6f7279020001"].to_vec(),
```

Comment:
The only remark I would like to make is why here square brackets are used but the usual brackets are used above?

Reply:
Good question. It's done this way in Substrate's own `chain_spec.rs` so I just blindly followed this pattern. But in most other places the round brackets are used. If it bothers you that much you can align it to some "canonical" style everywhere, I guess )

Reply:
I vote for round brackets as this macro is function-like. Square brackets are acceptable in `vec!` macro only in my opinion.

Reply:
Agree with circle brackets

Reply:
Fixed